### PR TITLE
COMMERCE-9433 Default display page template for a product

### PR DESCRIPTION
**Pull request by alessiorendina**
Changed after Brian Chan's review here: https://github.com/brianchandotcom/liferay-portal/pull/130091

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/display_layout/edit_asset_category_cp_display_layout.jsp [hunk 1]
```diff
@@ -31,14 +31,18 @@ if (cpDisplayLayout != null) {
...
 
 if (cpDisplayLayout != null) {
-	Layout selLayout = LayoutLocalServiceUtil.fetchLayoutByUuidAndGroupId(cpDisplayLayout.getLayoutUuid(), commerceChannel.getSiteGroupId(), false);
+	String layoutUuid = cpDisplayLayout.getLayoutUuid();
 
-	if (selLayout == null) {
-		selLayout = LayoutLocalServiceUtil.fetchLayoutByUuidAndGroupId(cpDisplayLayout.getLayoutUuid(), commerceChannel.getSiteGroupId(), true);
-	}
+	if (Validator.isNotNull(layoutUuid)) {
+		Layout selLayout = LayoutLocalServiceUtil.fetchLayoutByUuidAndGroupId(layoutUuid, commerceChannel.getSiteGroupId(), false);
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/display_layout/edit_asset_category_cp_display_layout.jsp:37]: Can we move this logic to a new displayContext method?

File: modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/layout_page_template_entry_item_selector.jsp [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+				).put(
+					"name", layoutPageTemplateEntry.getName()
+				).build());
+			%>
+
+			<liferay-ui:search-container-column-text
+				cssClass="table-cell-expand"
+				name="name"
+			>
+				<div class="commerce-product-definition-title" data-id="<%= layoutPageTemplateEntry.getLayoutPageTemplateEntryId() %>">
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-item-selector-web/src/main/resources/META-INF/resources/layout_page_template_entry_item_selector.jsp:58]: this class is wrong

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java [hunk 3]
```diff
@@ -337,6 +337,12 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 			"4.0.1", "4.0.2",
 			new CommerceRepositoryUpgradeProcess(_companyLocalService));
 
+		registry.register(
+			"4.0.2", "5.0.0",
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/registry/CommerceProductServiceUpgradeStepRegistrator.java:341]: this should be a minor change

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
see comments

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:commerce

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by alessiorendina** [comment]
ci:test:sf

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:test:commerce

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
ci:stop

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:stop

**Comment by alessiorendina** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**